src/back/glsl/features.rs

```diff
@@ -206,7 +206,12 @@ impl<'a, W> Writer<'a, W> {
     /// [`Error::MissingFeatures`](super::Error::MissingFeatures) will be returned
     pub(super) fn collect_required_features(&mut self) -> BackendResult {
         if let Some(depth_test) = self.entry_point.early_depth_test {
-            self.features.request(Features::IMAGE_LOAD_STORE);
+            // If IMAGE_LOAD_STORE is supported for this version of GLSL
```

Comment:
let's just remove this check

Reply:
On platforms where it is supported, we do check for the flag when writing out the extensions needed by the shader, does that mean we would still need the that?

Reply:
sounds reasobable